.github/workflows/aws-marketplace-camunda-helm-chart.yaml

```diff
@@ -0,0 +1,70 @@
+name: "AWS Marketplace Mirror: Camunda Helm chart"
```

Comment:
The prefix should have a separator like: `AWS Marketplace | Mirror: Camunda Helm chart`

Reply:
I added separators